README.md

```diff
@@ -1,9 +1,24 @@
 # Mutual Aid NYC Services List
 A [Next.js](https://nextjs.org) website for displaying NYC health, human and social services lists in the [Open Referral](https://openreferral.org) data format. Data is managed via [Airtable](https://airtable.com).
 
-## Tech Stack
-- [Chakra UI](https://chakra-ui.com) 
+## Tecnical Overview
+As of our most recent design, the major components and technologies are as follows: 
+* Website Front-End
+  * Uses Next JS and houses most (if not all) critical logic for fetching and displaying list data from the Intermediate Database. 
+* Intermediate Database
+  * Uses Postgress SQL and is provisioned by Sync Inc service for Airtable. Read-only(?) and is queried by the Front-End to avoid the Airtable API Query limits. 
```

Comment:
Based on [Sync Inc's docs](https://docs.syncinc.so/get-started#-optional-writing-to-airtable), the SQL DB itself will be read-only and we'll have to write to the Airtable which will trickle-down to the SQL